tests/data/car.py

```diff
@@ -0,0 +1,71 @@
+"""Copyright 2019 Equinor ASA and The Netherlands Organisation for
+Applied Scientific Research TNO.
+
+Licensed under the MIT license.
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the conditions stated in the LICENSE file in the project root for
+details.
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+"""
+
+
+from configsuite import MetaKeys as MK
+from configsuite import types
+
+
+def build_schema():
```

Comment:
I would really like to test the `Default`-values also for `NamedDicts` inside lists and dicts. As this is the real purpose of them in the schema anyway. You can either extent this schema, or try to find a another case in the test data that already features this...

Reply:
Extended the schema, but I really don't have any strong preferences.. I didn't want to change existing schemas when I made the PR, but it's possible to make use of some of the other ones.. If we find that there's too many data sets then I'll happily put the same setup in some existing schema